config/__init__.py

```diff
@@ -59,7 +59,10 @@ def initialize_status():
     finally:
         # save in data/ the current version
         if milestones_content:
-            app.config['STATUSES'] = convert_milestones(milestones_content)
+            updated_statuses = update_status_config(milestones_content)
+            if not updated_statuses:
+                return False
+            app.config['STATUSES'] = updated_statuses
```

Comment:
But if `updated_statuses` returns early, we still end up with `sys.exit('Milestones are not initialized.')`, right?